test/support/flow_test_helper.rb

```diff
@@ -71,8 +71,8 @@ def assert_rendered_question
   def assert_rendered_outcome(text: nil)
     ensure_valid_and_correct_node
 
-    assert_not_empty test_flow.outcome_body
-    assert_match text, test_flow.outcome_body_text if text
+    assert_not_empty test_flow.outcome_text
+    assert_match text, test_flow.stripped_outcome_text if text
```

Comment:
This looks good 👍 

I imagine if needed we could also have optional arguments like `assert_rendered_outcome(title: /bah/)` or `def assert_rendered_outcome(body_html: /<a href/)` if we wanted to do more granular assertions in a shorthand syntax.

Reply:
Yeah, I think this is best left for when there is a known need for it.